lib/rhc/commands/ssh.rb

```diff
@@ -15,6 +15,10 @@ class Ssh < Base
 
       You may run a specific SSH command by passing one or more arguments, or use a
       different SSH executable or pass options to SSH with the '--ssh' option.
+
+      To use the '--ssh' flag with an SSH executable path containing a space, wrap
+      the executable path with double quotes:
+      --ssh '"path with spaces\\ssh"'
```

Comment:
Either
--ssh '"/path with spaces/ssh"' or
--ssh '"C:\\path with spaces\ssh"' if you mean Windows